repeat_a_string_repeat_a_string.js

```diff
@@ -0,0 +1,12 @@
+function repeatStringNumTimes(str, num) {
+  
+  if(num <= 0) {
+    str = "";
```

Comment:
staraj sie uzywac guard pattern. https://en.wikipedia.org/wiki/Guard_(computer_science)
oznacza to, ze powinnas wychodzic z funkcji mozliwie wczesnie, jezeli warunki nie sa spelnione lub gdy mozesz podac rezultat.

w tym wypadku moglas tutaj dac return ''; a nizej procedowac bez else.